tests/end-to-end/cli/columns-max.phpt

```diff
@@ -8,6 +8,14 @@ $_SERVER['argv'][3] = __DIR__ . '/../../_files/BankAccountTest.php';
 
 require __DIR__ . '/../../bootstrap.php';
 PHPUnit\TextUI\Command::main();
+--SKIPIF--
+<?php
+
+if (PHP_OS_FAMILY !== 'Windows') {
+    die('Maximum column length resolves to different value on Windows systems');
```

Comment:
Perhaps 120 characters?

Reply:
Might be solved by 82a15e69b2088822b3e8afac24e21ff611b8fa53.

Reply:
Because only the `--prefer-lowest` builds were failing.

Reply:
120 is the default, and probably safe for CI although it is configurable on a developer system

![image](https://user-images.githubusercontent.com/1571110/86006980-b3f97500-ba0e-11ea-90de-7af6a92d7e96.png)

Reply:
That's it, @dvdoug and @sebastianbergmann!

👍